src/data/models/Vote.js

```diff
@@ -0,0 +1,36 @@
+import DataType from 'sequelize';
+import Model from '../db';
+import User from './User';
+import Submission from './Submission';
+
+const Vote = Model.define(
+  'vote',
+  {
+    id: {
+      type: DataType.UUID,
+      defaultValue: DataType.UUIDV1,
+      primaryKey: true,
+    },
+    comment: {
+      type: DataType.TEXT(),
+      allowNull: true,
+    },
+    result: {
+      type: DataType.ENUM('accepted', 'rejected'),
+    },
+  },
+  {
+    indexes: [
+      {
+        unique: true,
+        fields: ['submissionId', 'userId'],
+      },
+    ],
+  }
+);
+
+Submission.hasMany(Vote, { as: 'Votes' });
+Vote.belongsTo(Submission);
+Vote.belongsTo(User);
```

Comment:
It is better to move these to src/data/models/index.js when every table has been defined. We could run into errors if they are loaded in the wrong order.